modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/service/util/BaseWorkflowMetricsIndexerTestCase.java

```diff
@@ -226,6 +226,26 @@ protected void assertSLAReindex(
 		assertSLAReindex(indexNamesMap, indexTypes, parameters);
 	}
 
+	protected KaleoTaskInstanceToken assignKaleoTaskInstanceToken(
```

Comment:
Hello @rafaprax,

I put `getKaleoTaskInstanceTokens` method this way, what do you think?

Reply:
It's ok to me